dnf/repo.py

```diff
@@ -459,7 +459,7 @@ class Repo(dnf.conf.RepoConf):
     def __init__(self, name=None, parent_conf=None):
         # :api
         super(Repo, self).__init__(section=name, parent=parent_conf)
-        self._repofile = None
+        self.repofile = None
```

Comment:
We are prefixing all variables and methods that are not part of documented api by an underscore.